invenio_app_ils/records/api.py

```diff
@@ -407,12 +390,15 @@ def __init__(self, id, revision_id=None):
         self._profile = UserProfile.get_by_userid(id)
         self.name = self._profile.full_name if self._profile else ""
         self.email = self._user.email
+        # add all fields to the dict so they can be accessed as a dict too
+        super().__init__(self.dumps())
```

Comment:
@zzacharo I have changed the `Patron` class to use it as a dict too, so basically you can do:
`patron.id` or `patron["id"]`

This is because some code around that is generic and accept records was trying to do `record["id"]` and failing in case of patrons. Do you see any problem with this?

Reply:
No I think it should be fine :)